### ISSUE TITLE
Can't get dateTimeDigitized and dateTimeOriginal from image taken by Nokia Lumia

### ISSUE THREAD

**Issue by nokeya** [emmanueltouzery/hsexif#9]
Here is the image taken by Nokia Lumia 920 (on WP 8.1):
![_sync_nokia](https://cloud.githubusercontent.com/assets/3000370/9980922/4e2bb260-5fb4-11e5-965f-ff8f5a6f0ad1.jpg)

Other tools like exiftags, exiftime, exif show that it has 0x9003 (dateTimeOriginal) and 0x9004 (dateTimeDigitized) tags and their correct values.
When I'm trying to get its time and date with HsExif I get nothing.
Example code:
```haskell
import Data.Maybe(fromJust)
import qualified Data.Map as Map
import Graphics.HsExif
import System.Environment 

main = do
    args <- getArgs 
    img <- parseFileExif $ args !! 0
    let time = case img of
            Left _ -> error "unable to parse data"
            Right val -> fromJust $ Map.lookup dateTimeDigitized val
    print time
```

**Comment by emmanueltouzery** [comment]
interesting. If I open the jpg in gimp and save it back, then hsexif parses it ok. It seems hsexif only finds the tags in the root IFD, excluding the ones from the exif subifd and the gps subifd. It seems the other programs also skip the gps subifd though.

Looking into it.

**Comment by emmanueltouzery** [comment]
ok I believe I fixed the problem and I also uploaded 0.6.0.6 to hackage, which has the fix. Please reopen if it does not solve your issue!

**Comment by nokeya** [comment]
Thanks!:)
